internal/provider/provider_test_utils.go

```diff
@@ -47,11 +49,14 @@ func TestAccPreCheck(t *testing.T) {
 	}
 }
 
-func ProviderConfig(t *testing.T, isHosted bool) string {
+func ProviderConfig(t *testing.T, isHosted bool, isScim bool) string {
 	var orgId, token string
 	if isHosted {
 		orgId = os.Getenv("HOSTED_ORGANIZATION_ID")
 		token = os.Getenv("HOSTED_ORGANIZATION_API_TOKEN")
+	} else if isScim {
```

Comment:
actually it may be best to make this an enum that takes in `HOSTED` `SCIM` or `HYBRID`
so something like `ProviderConfig(t *testing.T, testOrganizationType string)` or `ProviderConfig(t *testing.T, testOrganizationType TestOrganizationType)`